### ISSUE TITLE
[Feature Request] Domain Categorization Improvements

### ISSUE THREAD

**Issue by ad0nis** [GhostManager/Ghostwriter#236]
**Is your feature request related to a problem? Please describe.**
Documentation (https://www.ghostwriter.wiki/features/infrastructure-management/domains-management/monitoring-domains) lists that categorizations are stored in "VENDOR": "CATEGORY" format. It would be great to be able to see this information within the domains tab so that operators know which vendor has which categorization, so they know where to go to try to change it, or can otherwise know the source of the judgement.

Additionally, it appears that there are some categories which should obviously lead to a flag of "burned", but don't appear to do so, such as "malware repository", or "spyware and malware". It might be nice to have this category list in the UI as a list that can be modified by admins instead of hardcoded in a python script.

**Describe the solution you'd like**
I would like for the vendor information to come through along with the categories.

Additionally, I'd like the list of categories that lead to a domain being burned to be an admin UI item instead of hardcoded to allow more flexibility.

**Describe alternatives you've considered**
Not sure how else I'd get this... Perhaps its in the database somewhere?

As far as the second half of the request, pretty clearly I can update the list myself. 

**Additional context**
N/A

**Comment by chrismaddalena** [comment]
Hey @ad0nis,

You can view categorization under the "Health" tab when viewing a domain's details. It will look something like this:

![image](https://user-images.githubusercontent.com/10526228/196823566-359f3a2d-c11e-4271-a940-fe053f623603.png)

Are you looking for it in a specific spot elsewhere in the UI?

Having the list of "bad" categories configurable is a good idea. The list is currently managed here:

https://github.com/GhostManager/Ghostwriter/blob/master/ghostwriter/modules/review.py#L46

Are there other potential categories you've seen applied to a domain not listed here? Vendors can get creative with the naming. The current list includes all the categories I've personally seen applied to known-bad domains in VirusTotal + categories many web proxies will generally ban or block:

```
        "phishing",
        "web ads/analytics",
        "suspicious",
        "placeholders",
        "pornography",
        "spam",
        "gambling",
        "scam/questionable/illegal",
        "malicious sources/malnets",
```

I can add "malware repository" and "spyware and malware" to the list.

**Comment by chrismaddalena** [comment]
Closing this for inactivity, and I believe the request was addressed in release v3.1.2.
